### PR TITLE
Improve tests reliability by returning included block

### PR DESCRIPTION
**Pull request by Thoralf-M**
# Description of change

Improve tests reliability by returning included block

## Type of change

- Enhancement (a non-breaking change which adds functionality)

## How the change has been tested

Running tests

### CODE REVIEW

**Review by thibault-martinez — DISMISSED** [review]

File: sdk/tests/client/node_api/mod.rs [hunk 1]
```diff
@@ -96,9 +96,9 @@ pub async fn setup_transaction_block(client: &Client) -> (BlockId, TransactionId
         _ => unreachable!(),
     };
 
-    let _ = client.retry_until_included(&block.id(), None, None).await.unwrap();
+    let retried = client.retry_until_included(&block.id(), None, None).await.unwrap();
```
Comment (Alex6323) [inline sdk/tests/client/node_api/mod.rs]: I know it's just for testing ... but the binding name is a bit lame .... `retried_block` or `retried_transaction`?

Comment (Thoralf-M) [reply sdk/tests/client/node_api/mod.rs]: Can be more than one, `retried_blocks`? Then please do a code suggestion and just merge

Comment (Alex6323) [reply sdk/tests/client/node_api/mod.rs]: Gh doesn't let me create a code suggestion for L101 :crying_cat_face:

Comment (thibault-martinez) [reply sdk/tests/client/node_api/mod.rs]: Changed

**Review by thibault-martinez — APPROVED** [review]

**Review by Alex6323 — APPROVED** [review]

**Pull request merged**